### ISSUE TITLE
Talkback lag

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#356]
```
What steps will reproduce the problem?
1. drawing of the angle gestures 
2. scrolling search results in google now 
3. opening or interacting with certain apps 

What is the expected output? What do you see instead?
Ideally talkback should begin speaking when I place my finger on screan to 
explore or swipe. Currently talkback hangs and I have found no consistent way 
to bring it back other than to wait, which can take anywhere from half a minute 
to five.

What version of the product are you using? On what operating system?
talkback 3.5.1 on android 4.1.2

Please provide any additional information below.
If this only happened in one app, I would just assume it was the app giving me 
problems, but there are several apps where I can explore to an item and 
talkback will give me no feedback for several minutes. This is also the case if 
I search for something with google now.  While it is worse on my phone--nearly 
impossible most times--it does happen on my tablet as well. Both are running 
the same versions of talkback and android. And if I should try swiping while I 
am waiting for talkback to speak, I can hear the swipe sounds, but almost as if 
talkback is streaming and having trouble buffering. The same is true of the 
angle gestures.  I hear the sound, so I know I've done the gesture 
successfully, but it takes talkback a minute to respond when I try to draw the 
circle.
I'd be very greatful if this could be fixed, as it is really my only issue with 
talkback.
Thanks for everything!

```

Original issue reported on code.google.com by `sculpeyg...@gmail.com` on 23 Feb 2014 at 10:34
